src/pages/Shopping/shoppingList.js

```diff
@@ -40,19 +64,39 @@ export class shoppingList extends Component {
             <option value="기타">리뷰순</option>
           </select>
         </form>
-
         <section className="itemContainer">
           <ul className="items">
-            <Item />
-            <Item />
-            <Item />
-            <Item />
-            <Item />
+            {this.state.items.map(item => {
+              if (
+                undefined === this.props.match.params.name ||
+                'allmeat' === this.props.match.params.name
+              ) {
+                return (
+                  <Item
+                    key={item.title}
+                    id={item.id}
+                    img={item.img}
+                    price={item.price}
+                    title={item.title}
+                  />
+                );
+              } else if (item.category === this.props.match.params.name) {
+                return (
+                  <Item
+                    key={item.title}
+                    id={item.id}
+                    img={item.img}
+                    price={item.price}
+                    title={item.title}
+                  />
+                );
+              }
+            })}
```

Comment:
- 비교 대상이 왼쪽, 비교 하는 "값" 이 오른쪽에 위치하는게 일반적 입니다!
- `if (this.props.match.params.name === undefined) ` 와 같이 작성해주세요!
- 비교하는 대상들을 배열에 담은 뒤, includes 메서드를 사용하면 더 간결하고 읽기 쉬운 코드를 작성하실 수 있을 것 같습니다.
```suggestion
            {this.state.items.map(item => {
              const allowedPath = [undefined, 'allmeat', item.category];
              if (allowedPath.inclues(this.prop.smatch.params.name) {
                return (
                  <Item
                    key={item.title}
                    id={item.id}
                    img={item.img}
                    price={item.price}
                    title={item.title}
                  />
                );
              }
            })}
```

Reply:
이런방법도 있군요! 감사합니다!!👍